src/codegen/codegen_acc_visitor.cpp

```diff
@@ -124,6 +136,18 @@ void CodegenAccVisitor::print_net_send_buffering_grow() {
     // can not grow buffer during gpu execution
 }
 
+void CodegenAccVisitor::print_eigen_linear_solver(const std::string& float_type,
+                                                  int N,
+                                                  const std::string& Xm,
```

Comment:
```suggestion
                                                  const std::string& X,
```
Should this be like this to follow the declaration of the function in the header file or change the name of the variable in the declaration?